Makefile

```diff
@@ -64,6 +64,17 @@ ifdef V8_SRCDIR
 override CPPFLAGS += -I$(V8_SRCDIR) -I$(V8_SRCDIR)/include
 endif
 
+ifeq ($(OS),Windows_NT)
```

Comment:
i hadn't heard of any compilation errors except on OS X (i don't have mingw, even in a vm).

i the check there just in case there are flags that need to be added for windows environments.  for now, since there weren't any, i left it as a no-op.